sql/src/main/java/org/apache/druid/sql/calcite/planner/JoinHint.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.sql.calcite.planner;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import org.apache.druid.java.util.common.IAE;
+import org.apache.druid.query.JoinAlgorithm;
+
+import java.util.Arrays;
+
+public enum JoinHint
+{
+  USE_MERGE_JOIN("use_merge_join") {
```

Comment:
why the `use_` and `no_` prefixes?
I feel like it would be better to not overthink it for now.... we could have `broadcast` and `sort_merge` to align with the internal logic...
I think its better to not create alternates if there is no specific need for it.

Reply:
updated